libs/application/template-api-modules/src/lib/modules/templates/new-primary-school/new-primary-school.service.ts

```diff
@@ -38,6 +40,12 @@ export class NewPrimarySchoolService extends BaseTemplateApiService {
   }
 
   async getChildren({ auth }: TemplateApiModuleActionProps) {
+    if (!isRunningInProduction) {
```

Comment:
Should we not use a feature flag for flow control?

Reply:
The application is under a feature flag and is not currently in use on prod. We are just adding this to solve the issue with Þjóðskrá not having children for these test users. This is just for Júní so they can demo the application in dev environment